core/templates/dev/head/pages/profile/profile.html

```diff
@@ -134,12 +134,13 @@ <h2 class="oppia-profile-username-small-screen">
       </md-content>
 
       <span class="oppia-profile-portfolio-pages" ng-if="numUserPortfolioExplorations > 6">
-        <i class="material-icons md-18" ng-disabled="currentPageNumber == 0"
+        <i ng-class="{'material-icons md-18 oppia-navigation-button':true,
+                      'disabled':startingExplorationNumber==1}"
```

Comment:
I agree with @hoangviet1993, better to do ng-disabled.

Reply:
Done.!